src/Service/Azure.DataApiBuilder.Service.csproj

```diff
@@ -28,7 +29,7 @@
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Docker|AnyCPU'">
-      <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+        <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
```

Comment:
nit: remove trailing space

Reply:
The spacing was incorrect before. This is correct spacing (4 spaces)